packages/cogutil/cogutil.opam-1.0/opam

```diff
@@ -0,0 +1,29 @@
+(** OPAM (OCaml Package Manager) module for CogUtil *)
+(** *)
+(** This is in the root directory because I cannot figure out *)
```

Comment:
That is correct, the opam files are in the root of ocaml packages distributed via opam. But this seems to be a C/C++ package, so I am not sure why you need the opam file at all